[docs] Extend the glossary (#245)
hyperledger/iroha-2-docs: src/guide/glossary.md

- [World state view (WSV)](#world-state-view-wsv)
- [Leader](#leader)

## Blockchain ledgers

Blockchain ledgers are digital record-keeping systems that use blockchain technology to keep financial records.

> These are named after old-fashioned books used for financial records[^1] because of their history.

Reviewer: ```suggestion
These are named after old-fashioned books that were used for financial records[^1] because of their history.
```

Reviewer 2: ```suggestion
These are named after old-fashioned books that were used for financial records such as prices, news, and transaction information.
```

During medieval times, ledger books were open for public viewing and accuracy verification.

> This idea found its reflection in the blockchain-based systems that can check the stored data for validity.

Reviewer: ```suggestion
This idea is reflected in the blockchain-based systems that can check the stored data for validity.
```

## Asset

> While the word itself means a valuable object, with blockchain being involved, we'll be talking about a representation of a valuable object on the blockchain.

Reviewer: ```suggestion
While the word itself means "a valuable object," with blockchain being involved, we are referring to the representation of a valuable object on the blockchain.
```

Reviewer 2: ```suggestion
In the context of blockchains, an asset is the representation of a valuable object on the blockchain.
```

Additional information on assets is available [here](/guide/blockchain/assets.md).

### Fungible assets

> Such assets can be easily swapped with other assets of the same type, because they are interchangeable.

Reviewer: ```suggestion
Such assets can be easily swapped for other assets of the same type because they are interchangeable.
```
